assembl/models/idea_content_link.py

```diff
@@ -50,7 +50,7 @@ class IdeaContentLink(DiscussionBoundBase):
 
     # This is nullable, because in the case of extracts, the idea can be
     # attached later.
-    idea_id = Column(Integer, ForeignKey(Idea.id),
+    idea_id = Column(Integer, ForeignKey(Idea.id, ondelete="CASCADE", onupdate="CASCADE"),
```

Comment:
Doesn't this have to be defined in a migration as well to be a part of the sql schema?

Reply:
There may be a migration missing yes. Can you add it? The change here was mainly to fix the tests. In reality ideas are not removed from the database.

Reply:
Yeah, I'll add a migration for this.